src/components/GlobalStyles.js

```diff
@@ -0,0 +1,32 @@
+import React from "react";
+import { Global, css } from "@emotion/core";
+
+export default function GlobalStyles() {
+  return (
+    <Global
+      styles={css`
+        *,
+        *::after,
+        *::before {
+          box-sizing: border-box;
+        }
+        body {
+          margin: 0px;
+          background: #2b2758;
+          font-family: "News Cycle", sans-serif;
+          color: white;
+        }
+        p {
```

Comment:
we won't set styles for elements in the global styles. instead, emotion/jss will take care of it in the components.